### PR TITLE
Bump version to 1.9.6-alpha

### PR DESCRIPTION
**Pull request by bwateratmsft**
Bumps version to 1.9.6-alpha, along with dependencies. Please don't merge yet.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2778,17 +2778,17 @@
...
         "@azure/arm-appservice": "^6.1.0",
         "@azure/arm-containerregistry": "^8.0.0",
         "@azure/storage-blob": "^12.3.0",
         "@docker/sdk": "^1.0.3",
-        "@grpc/grpc-js": "^1.2.2",
-        "dayjs": "^1.10.2",
-        "dockerfile-language-server-nodejs": "^0.2.1",
+        "@grpc/grpc-js": "^1.2.3",
+        "dayjs": "^1.10.3",
+        "dockerfile-language-server-nodejs": "^0.2.2",
```
Comment (rcjsuen) [inline package.json:2791]: Hopefully this will help with #2601.

Comment (bwateratmsft) [reply package.json:2791]: @rcjsuen Should we just grab 0.2.2 for our 1.9.1 release (~Monday)?

Comment (rcjsuen) [reply package.json:2791]: @bwateratmsft It shouldn't hurt. The actual code changes are [minimal](https://github.com/rcjsuen/dockerfile-language-server-nodejs/compare/v0.2.1...v0.2.2), just implementing what was suggested in https://github.com/microsoft/vscode-docker/issues/2601#issuecomment-756134327.

Has anyone else hit #2601? The stack trace indicates it was reproduced on 1.9.5-alpha. 🤔

Comment (bwateratmsft) [reply package.json:2791]: I've got it showing up on our 1.9.1 candidate build. I'll make a PR and we can discuss in our team triage today.

File: package-lock.json [hunk 2]
```diff
@@ -4808,6 +4818,15 @@
                         "read-pkg": "^5.2.0",
                         "type-fest": "^0.8.1"
                     }
+                },
+                "semver": {
```
Comment (karolz-ms) [inline package-lock.json:4822]: 😨

**Review by karolz-ms — APPROVED** [review]
Just double-check please that the webpacked extension loads after this change

File: package-lock.json (hunk 2 above)
Comment (bwateratmsft) [reply package-lock.json:4822]: Absolutely! I think that this problem is actually why semver was tripping in the first place--there was one version webpacked in, and another included as a dependency of the language client.

Comment (bwateratmsft) [reply package-lock.json:4822]: Aha, that was _exactly_ it! And now fixed in #2626.

**Pull request merged**